Gems/AudioSystem/Code/Source/Engine/ATLAudioObject.cpp

```diff
@@ -1042,7 +1042,8 @@ namespace Audio
         auxGeom.SetRenderFlags(newRenderFlags);
 
         const bool drawRays = CVars::s_debugDrawOptions.AreAllFlagsActive(DebugDraw::Options::DrawRays);
-        const bool drawLabels = CVars::s_debugDrawOptions.AreAllFlagsActive(DebugDraw::Options::RayLabels);
+        // ToDo: Update to work with Atom? LYN-3677
+        //const bool drawLabels = CVars::s_debugDrawOptions.AreAllFlagsActive(DebugDraw::Options::RayLabels);
```

Comment:
Where did this ticket come from?

Reply:
from this same file below, the code that is using `drawLabels` is commented with the same comment

Reply:
Thanks for the clarification.